docs/index.md

```diff
@@ -0,0 +1,70 @@
+# Iterative Provider
+
+Use the Iterative Provider to launch resource-intensive tasks in popular cloud
+providers with just a simple Terraform file.
+
+## Example Usage
+
+```hcl
+terraform {
```

Comment:
This section shows a sample config but not it's usage. Should we at least also mention `terraform init/apply`?
And/or link to the full Get Started guide.

Reply:
p.s. this may apply to docs/resources/task.md additionally (or instead).

Reply:
Popular providers' documentation uses the `index.md` **Example Usage** section to target those users that know how to use Terraform and just need a bit of boilerplate to get started.

New users should follow the **Getting Started** guide instead. Still, we have code snippets in three different places; not sure if it's a good idea.

Reply:
https://github.com/iterative/terraform-provider-iterative/pull/237/files/739955bf1943bb25a1d3711ce232f4d5ff67676b#r749787219